tools/releaseBuild/Image/DockerFile

```diff
@@ -14,19 +14,20 @@ COPY dockerInstall.psm1 containerFiles/dockerInstall.psm1
 RUN Import-Module PackageManagement; `
     Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force; `
     Import-Module ./containerFiles/dockerInstall.psm1; `
+    [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor [System.Net.SecurityProtocolType]::Tls12; `
     Install-ChocolateyPackage -PackageName git -Executable git.exe; `
     Install-ChocolateyPackage -PackageName nuget.commandline -Executable nuget.exe -Cleanup; `
     Install-Module -Force -Name platyPS -Repository PSGallery; `
     Invoke-WebRequest -Uri https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.ps1 -outfile C:/dotnet-install.ps1; `
     C:/dotnet-install.ps1 -Channel Release -Version 2.1.4; `
     Add-Path C:/Users/ContainerAdministrator/AppData/Local/Microsoft/dotnet;
 
-RUN Import-Module ./containerFiles/dockerInstall.psm1; `
+#RUN Import-Module ./containerFiles/dockerInstall.psm1; `

```

Comment:
Might be worth removing this entirely if we're no longer using this

Reply:
sure